sr_fingertip_visualization/src/sr_fingertip_visualization/finger_widgets_graphs.py

```diff
@@ -252,3 +254,53 @@ def start_timer_and_subscriber(self):
 
     def stop_timer_and_subscriber(self):
         pass
+
+
+class FingerWidgetGraphMSTBlank(QGroupBox):
+    def __init__(self, side, parent):
+        super().__init__(parent=parent)
+        self._side = side
+
+        self.setTitle("---")
+        self.setCheckable(True)
+        self.setChecked(True)
+        self.setSizePolicy(1, 1)
+
+        layout = QVBoxLayout()
+
+        layout.setAlignment(Qt.AlignVCenter)
+        no_tactile_label = QLabel("\t\tSTF sensors not supported yet."
+                                  " \nAlternatively, you can launch the RViz or PlotJuggler visualization tools:")
+        layout.addWidget(no_tactile_label, alignment=Qt.AlignCenter)
+
+        self.launch_viz_button = QPushButton("Launch RViz")
+        self.launch_viz_button.clicked.connect(self._button_action_launch_viz)
+        layout.addWidget(self.launch_viz_button, alignment=Qt.AlignCenter)
+
+        self.launch_plotjuggler_button = QPushButton("Launch PlotJuggler*")
+        self.launch_plotjuggler_button.clicked.connect(self._button_action_launch_plotjuggler)
+        layout.addWidget(self.launch_plotjuggler_button, alignment=Qt.AlignCenter)
+
+        note_label = QLabel("*Note: You'll need to click 'Yes' to Start the Streaming and select"
+                            f" /{self._side}/tactile data topic.")
+        layout.addWidget(note_label, alignment=Qt.AlignCenter)
+
+        self.setLayout(layout)
+
+    def _button_action_launch_viz(self):
+        command = f"roslaunch sr_mst sr_mst_hand_rviz_visualiser.launch hand_id:={self._side} publishing_frequency:=30"
+        rospy.loginfo(f"Launching RViz for visualization of STF fingertips: {command}")
+        subprocess.Popen(command, shell=True)  # pylint: disable=R1732
```

Comment:
Is this best practise? seems a bit sketchy

Reply:
Changed now to use roslaunch api